adapters/ion-adapter/src/main/java/com/connexta/ion/replication/adapters/ion/IonNodeAdapterFactory.java

```diff
@@ -58,7 +58,6 @@ public IonNodeAdapterFactory(
 
   @Override
   public NodeAdapter create(URL url) {
-    // TODO change this when ingest enpoint supports https
```

Comment:
❓ Why is this no longer a todo?

Reply:
It was getting flagged by sonar cloud and there is nothing we could do about it. When the ingest endpoint updates to use https they will notify us and we can make the change then. Don't think we need to the todo to track that.